charts/argo-cd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.7.11
 description: A Helm chart for ArgoCD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 2.11.7
+version: 2.11.8
```

Comment:
Considering that you're adding a new feature to the chart (the ability to add lifecycle hooks), I guess you should increase its minor version (2.12.0).

Reply:
@yann-soubeyrand right you are! pushing the minor up